cobra/util/solver.py

```diff
@@ -337,18 +338,32 @@ def add_absolute_expression(model, expression, name="abs_var", ub=None):
        The name of the newly created variable.
     ub : positive float
        The upper bound for the variable.
+    difference : positive float
+        The difference between the expression and the variable.
+    add : bool
+        Whether to add the variable to the model at once.
+
+    Returns
+    -------
+    list
```

Comment:
how about making the return value a `namedtuple('Components', ['variable', 'upper_constraint', 'lower_constraint'])` and avoid the need to remember the order they come? Tuples also work with `model.add_cons_vars`